doordeck-sdk/src/commonMain/kotlin/com/doordeck/multiplatform/sdk/model/data/LockOperations.kt

```diff
@@ -42,21 +42,21 @@ object LockOperations {
         val latitude: Double,
         val longitude: Double,
         val enabled: Boolean? = null,
-        val radius: Int? = null,
-        val accuracy: Int? = null
+        val radius: Int = 20,
+        val accuracy: Int = 20
     ) {
         class Builder {
             private var latitude: Double? = null
             private var longitude: Double? = null
             private var enabled: Boolean? = null
-            private var radius: Int? = null
-            private var accuracy: Int? = null
+            private var radius: Int = 20
+            private var accuracy: Int = 20
```

Comment:
We should probably use the same defaults as the backend (I know it's not strictly needed but seems safer), raduis should be 100 and accuracy 200.

Reply:
Done!